cmd/dbmigrate/utils/migrator.go

```diff
@@ -455,16 +460,19 @@ func (m *migrationManager) MigrateDBDir(dbDir string) (summary string, err error
 		}
 		// always wrap any error with some extra context.
 		if err != nil {
-			err = fmt.Errorf("could not convert %q from %q to %q: %w", dbDir, sourceDBType, m.TargetDBType, err)
+			err = fmt.Errorf("could not convert %q to %q: %w", dbDir, m.TargetDBType, err)
```

Comment:
Please keep the source db type in this error message. It's a crucial piece of info when troubleshooting. I know it might be in the log output somewhere, but it might not. And having it here means we don't have to dig through the logs for it.

Reply:
Added it back.